pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -343,6 +348,28 @@ func manageKubeAPIServerCABundle(lister corev1listers.ConfigMapLister, client co
 	return resourceapply.ApplyConfigMap(client, recorder, requiredConfigMap)
 }
 
+func syncTrustedCABundle(lister corev1listers.ConfigMapLister, client coreclientv1.ConfigMapsGetter, recorder events.Recorder) (bool, error) {
```

Comment:
where do you use the `bool` return value?

Reply:
I was originally merely following the pattern of the `manage*` function, I'll remove that as it does not feel necessary